src/bynder-js-sdk.js

```diff
@@ -597,4 +597,86 @@ export default class Bynder {
         );
         return request.send();
     }
+
+    /**
+     * Create the collection information according to the name provided.
+     * @see {@link http://docs.bynder.apiary.io/#reference/collections/specific-collection-operations/create-collection|API Call}
+     * @param {Object} queryObject={} - An object containing the id of the desired collection.
+     * @param {String} queryObject.name - The name of the desired collection.
+     * @param {String} queryObject.description - The description of the desired collection.
+     * @return {Promise} Collection - Returns a Promise that, when fulfilled, will either return an Object with the
```

Comment:
In this case what's return is just a `Response` object, right? I would change to `(...) return an object with the Response or (...)`.

Reply:
Indeed, Changed